Frontend/src/overrides.css

```diff
@@ -1,5 +1,9 @@
-/* We don't want to have the padding that is caused by having two containers */
-.registration-container {
+/* We don't want to have the padding that is caused by having two containers in admin mode */
+.container > #registration-app > .registration-container {
     padding-left: 0 !important;
     padding-right: 0 !important;
+}
+
+.competition-info-logo {
+    margin-top: 0 !important;
```

Comment:
Are you sure it's the *top* margin that needs to be fixed?
In my dev console at first glance it looks like the *bottom* margin is causing problems, which makes sense because the clickable icons on the bottom of the comp image appear shifted.

Reply:
You would think so, but look at this 
![margin-enabled](https://github.com/thewca/wca-registration/assets/10168980/c91474cd-31ac-4c4f-a8f2-ebf1bf756352)
![margin fixed](https://github.com/thewca/wca-registration/assets/10168980/5a426354-f990-4b2f-8b61-8b0551617af6)

Reply:
Ah, that's because in this _specifc_ case, removing the top margin shifts the image upwards enough to make sure that the bottom margin doesn't matter anymore.

I'm not sure whether that's a clean solution. The bottom margin is clearly the one causing the issue.
Can you ask ChatGPT about this? (In two vertically adjacent `div`s, there are elements floated to the right. The top one has... The bottom one becomes misaligned because... Should you fix this by removing the top row's content's margin-top or margin-bottom?)